.upptimerc.yml

```diff
@@ -25,9 +25,10 @@ sites:
   - name: Data portal Staging
     url: https://gui-staging.dandiarchive.org/
   - name: ETelemetry server
-    url: https://rig.mit.edu/et/
+    url: https://etelemetry.dandiarchive.org/
```

Comment:
is there a reason why the original url doesn't work given a CNAME redirect?

Reply:
thanks for digging @aaronkanzer . ok, let's proceed with this in mind for now. If you could subscribe to that issue and react if somehow gets resolved -- we might want to get back then

Reply:
Hmmm still noticing that the service is "Down" 🤔 

Going to continue to investigate....

Reply:
This was a fun one to find:

Seems that `upptime` defines health by a `HEAD` request and not a `GET` request:

```
aaronkanzer@Aarons-MacBook-Pro ~ % curl -I https://etelemetry.dandiarchive.org/et/
HTTP/2 405 
date: Tue, 25 Mar 2025 13:15:01 GMT
content-type: text/plain; charset=utf-8
content-length: 0
server: nginx/1.27.4
allow: GET

aaronkanzer@Aarons-MacBook-Pro ~ % curl -X GET -I https://etelemetry.dandiarchive.org/et/
HTTP/2 200 
date: Tue, 25 Mar 2025 13:15:09 GMT
content-type: application/json
content-length: 141
server: nginx/1.27.4
```

Commuting over to MIT now, but will push a change soon

Reply:
in that case, let's fix the etelemetry server to respond to a HEAD request and change this back to normal URL

Reply:
Provided https://github.com/dandi/upptime/pull/391 in response here